rootfs/api/models/certificate.py

```diff
@@ -179,15 +180,15 @@ def attach_in_kubernetes(self, domain):
             }
 
             secret = self._scheduler._get_secret(app, name).json()['data']
-        except KubeHTTPException:
+        except KubeException:
             self._scheduler._create_secret(app, name, data)
         else:
             # update cert secret to the TLS Ingress format if required
             if secret != data:
                 try:
                     self._scheduler._update_secret(app, name, data)
-                except KubeHTTPException:
-                    raise
+                except KubeException as e:
+                    raise ServiceUnavailable(str(e)) from e
```

Comment:
But this is in the certificate model that I am re-raising as a different type, not the scheduler. Can you elaborate what you mean?